cloudinit/sources/helpers/vmware/imc/guestcust_util.py

```diff
@@ -137,7 +137,7 @@ def get_tools_config(section, key, defaultVal):
     cmd = ['vmware-toolbox-cmd', 'config', 'get', section, key]
 
     try:
-        (outText, _) = util.subp(cmd)
+        (outText, _) = util.subp(cmd, rcs=[0, 69])
```

Comment:
I agree that it would be preferable to log exactly what's going on, for easier debugging in future. Is that something you're comfortable doing @chengcheng-chcheng?

Reply:
Thanks for your review. Have changed to log the contents.